<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>608</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>55</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>RANSAC</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget1" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="image1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>1</width>
           <height>1</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="load1">
         <property name="text">
          <string>Load first picture</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pictureInfo1">
         <property name="text">
          <string>no loaded</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadHaraff1">
         <property name="text">
          <string>Load first haraff.sift file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pictureHaraffInfo1">
         <property name="text">
          <string>no loaded</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="load2">
         <property name="text">
          <string>Load second picture</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pictureInfo2">
         <property name="text">
          <string>no loaded</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadHaraff2">
         <property name="text">
          <string>Load second haraff.sift file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="pictureHaraffInfo2">
         <property name="text">
          <string>no loaded</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="size_label">
           <property name="minimumSize">
            <size>
             <width>178</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>78</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="size">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>10</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="cohesion_label">
           <property name="minimumSize">
            <size>
             <width>178</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>178</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Cohesion</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="cohesion">
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="iterations_label">
           <property name="minimumSize">
            <size>
             <width>178</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>178</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Iterations</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="iterations">
           <property name="text">
            <string>10</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="error_label">
           <property name="minimumSize">
            <size>
             <width>178</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>178</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>Max error</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="error">
           <property name="text">
            <string>60</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QRadioButton" name="affine">
         <property name="text">
          <string>Affine transformation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="perspetcive">
         <property name="text">
          <string>Perspective Transformation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="start">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveImage">
         <property name="text">
          <string>Save image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="returnCode">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>726</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
